ui/modules/environments/environments.js

```diff
@@ -0,0 +1,131 @@
+/*
+ *
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+/* eslint-disable arrow-parens */
+/* eslint-disable prefer-const */
+/* eslint-disable no-invalid-this */
+/* eslint-disable require-jsdoc */
+import * as Utils from '../utils.js';
+import * as API from '../api.js';
+
+let environments = {
+  local_ditto: {
+    api_uri: 'http://localhost:8080',
+    solutionId: null,
+    bearer: null,
+    usernamePassword: 'ditto:ditto',
+    usernamePasswordDevOps: 'devops:foobar',
+    useBasicAuth: true,
+  },
+  eclipse_sandbox: {
+    api_uri: 'https://ditto.eclipseprojects.io',
+    bearer: '',
+    usernamePassword: null,
```

Comment:
we could also pre-fill `'ditto:ditto'` as usernamePassword here ..

Reply:
I adapted that. Btw. I was not able to test with the sandbox, because of CORS restrictions. May be we should fully remove the second environment until this is resolved? Or we could set a sample for IoT Things?

Reply:
Oh, agreed - remove the sandbox environment then.
The sandbox is currently not in a stable state anyways..

Reply:
ok, done. Thanks @thjaeckle